Created & Updated Carousel components and Footer

leovaldez08 opened the pull request:
Created the carousel components like Midjourney.jsx, ChatGpt.jsx and StableDiffusion.jsx. Created CardData files for the carousels. Created Footer section of the page. Also made minor changes in Navbar. Finished the home page.

leovaldez08 committed:
Created and Updated Carousel components, Footer

Created the carousel components like Midjourney.jsx, ChatGpt.jsx and StableDiffusion.jsx. Created CardData files for the carousels. Created Footer section of the page. Also made minor changes in Navbar. Finished the home page.

leovaldez08 committed:
Update README.md

Added PR links to the progress tracker

jithin-kalvian reviewed (changes requested):
Please complete the changes

yash0212 reviewed (changes requested):
Please resolve the comments below

leovaldez08 commented on FrontEnd/src/App.jsx:
Noted! Made changes

leovaldez08 commented on FrontEnd/src/Components/ChatGpt.jsx:
```diff
@@ -0,0 +1,25 @@
+        <Divider mb="16px" borderColor="gray.400" borderBottomWidth="1.5px" />
+      </Box>
+      <Box px="2%">
+        <Carousel cardData={cardData}></Carousel>
```
Got it! Made changes!

leovaldez08 commented on FrontEnd/src/Components/Carousel.jsx:
```diff
@@ -0,0 +1,81 @@
+/* eslint-disable react/prop-types */
```
Explained in the below comment

leovaldez08 commented on FrontEnd/src/Components/Footer.jsx:
```diff
@@ -0,0 +1,130 @@
+/* eslint-disable react/no-unknown-property */
+/* eslint-disable react/prop-types */
```
Both of them are inbuilt rules of es-lint.
The first line is used to disable the rule which throws an error when a component uses an unknown HTML attribute.
The second line is used to disable the rule which throws an error when a component doesn't use the prop types correctly. In my case, it refused to accept the prop.
I chose to disable the rules as a temporary fix.

leovaldez08 replied:
Resolved!

leovaldez08 commented on FrontEnd/src/Components/Carousel.jsx:
Understood! Added id to the CardData files and modified the map function!

leovaldez08 committed:
Updated app.jsx and CardData files

Resolved and Implemented the given PR feedbacks. Added unique keys to the card data files. Modified the map function in Carousel.jsx. Made all the home page components responsive using relative units.

yash0212 commented on FrontEnd/src/Components/HeroSection.jsx:
```diff
@@ -81,49 +85,53 @@ const HeroSection = () => {
             marginRight={{ base: 0, md: "28px" }}
             borderRadius="3px"
             _hover={{ color: "none" }}
+            // maxW={{ base: "100%", md: "100%" }}
```
Commented code can be removed

leovaldez08 replied:
Done!

yash0212 reviewed (changes requested):
Please format all the changed files and resolve the comments

leovaldez08 committed:
Created and updated ModalForm.jsx

Finished ModalForm component and made some minor changes in carousel and navbar

leovaldez08 committed:
changes the location of the gitignore file

leovaldez08 committed:
Added react prop-types

resolved the PR feedback

leovaldez08 committed:
Added react routes and Pages folder for routing

leovaldez08 committed:
Update HeroSection.jsx
